Makefile

```diff
@@ -45,9 +45,9 @@ ci:
 
 .PHONY: clean
 clean:
-	@rm -rf output && rm -rf fdp/graphdb && rm -f fdp/docker-compose.yml && rm -rf fdp/openrefine-cfg/*.yaml
+	@rm -rf output && rm -rf fdp/graphdb && rm -f fdp/compose.yml && rm -rf fdp/openrefine-cfg/*.yaml
```

Comment:
openrefine can be deleted here as well?

Reply:
Done. I also removed other remaining references to openrefine.